gulp-tasks/utils/external-helpers.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
```

Comment:
why disable eslint?

Reply:
I didn't follow why that was done here either.